apps/account/templates/a4_candy_account/account_deletion.html

```diff
@@ -0,0 +1,54 @@
+{% extends "a4_candy_account/account_dashboard.html" %}
+
+{% load i18n %}
+
+{% block title %}
+    {% translate 'Your user profile' %}
+    &mdash;
+    {{ block.super }}
+{% endblock %}
+{% block dashboard_content %}
+    <div class="my-5 my-md-0">
+        <h1 class="mt-0">{% translate 'Delete account' %}</h1>
+        <p class="u-text--gray">{% blocktranslate %}Please note that deleting your account will result in the permanent removal of all associated data and content, including your comments, ideas & votes.{% endblocktranslate %}</p>
+        <p class="u-text--gray">{% blocktranslate %}
+            To confirm your identity, please enter your password to proceed:{% endblocktranslate %}</p>
+        <form id="account-delete-form" novalidate action="{% url "account_deletion" %}" method="post">
+            {% csrf_token %}
+            {% for field in form %}
+                {% include 'a4_candy_contrib/includes/form_field.html' with field=field %}
+            {% endfor %}
+            {# prevent form submission via enter/return, effectively bypassing the modal #}
+            <button class="d-none" type="submit" disabled aria-hidden="true"></button>
```

Comment:
@hom3mad3 I realized that the form submits without the modal when you just press enter after entering the password. This seems to be a legit way to block that behavior, what do you think?